File: exercises_00/04.py
"""
    Program that:
        Recientemente se descubrieron 7 planetas similares a la Tierra y potencialmente
        habitables que orbitan alrededor de una estrella denominada TRAPPIST-1. El
        sistema solar se encuentra a aproximadamente 369 trillones de kilómetros (Un
        trillón equivale a 1018 kilometros). Asumamos que podemos viajar a la velocidad de
        la luz. ¿A cuántos años luz está ese sistema solar?
"""

LIGHT_YEAR: int = 9_460_730_472_580_800 # In meters
DISTANCE_EARTH_TRAPPIST: int = 369 * 1_018_000 # In meters

def main () -> None:
    print(f"TRAPPIST esta a {DISTANCE_EARTH_TRAPPIST / LIGHT_YEAR} años luz de la tierra")

if __name__ == '__main__':
    main()

File: c_exercises_00/04.c
/*
 * =====================================================================================
 *
 *       Filename:  04.c
 *
 *    Description: Recientemente se descubrieron 7 planetas similares a la Tierra y potencialmente
        habitables que orbitan alrededor de una estrella denominada TRAPPIST-1. El
        sistema solar se encuentra a aproximadamente 369 trillones de kilómetros (Un
        trillón equivale a 1018 kilometros). Asumamos que podemos viajar a la velocidad de
        la luz. ¿A cuántos años luz está ese sistema solar?
 
 *
 *        Version:  1.0
 *        Created:  08/30/2023 06:15:21 PM
 *       Compiler:  gcc
 *
 * =====================================================================================
 */

#include <stdio.h>

long double LIGHTYEAR = 9460730472580800.0; // In meters
long DISTANCE_EARTH_TRAPPIST = 369 * 1018000; // In meters

int main (void) {
  printf("%Lf", (DISTANCE_EARTH_TRAPPIST / LIGHTYEAR));
  return 0;
}
